apps/test/unit/lib/kits/maker/FakeChromeSerialPort.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file Stand-in for chrome-serialport library, for use in tests.
+ */
+class SerialPort {}
+
+let deviceList = [];
+
+export default {
+  SerialPort,
+
+  isInstalled(callback) {
+    callback(null);
+  },
+
+  // Give back a valid list of serial port info
+  list(callback) {
+    callback(null, deviceList);
+  },
+
+  test_setDeviceList(list) {
+    deviceList = list;
+  },
+
+  test_reset() {
+    deviceList = [];
+  }
+};
```

Comment:
According to Fowler [we're both wrong](http://stackoverflow.com/a/346440/5000129) - this is neither a fake nor a mock, it's a stub.

* Fakes are a "working implmentation," even if it's a minimal/shortcut one.  Example: [fakeredis](https://github.com/guilleiguaran/fakeredis)
* Mocks have expectations about the calls they receive. Example: [sinon mocks](http://sinonjs.org/releases/v1.17.7/mocks/)
* Stubs do nothing and tell you what you want to hear.  This is a stub.

I'll updating naming accordingly.  I like your `SerialPort.stub.setDeviceList()` pattern, I'll update to that.